models/MODEL_EDITING_TEST_SUITE/ONE_ASSIGNMENT_RULE/ONE_ASSIGNMENT_RULE-settings.txt

```diff
@@ -0,0 +1,8 @@
+start: 0
+duration: 0.1
+steps: 50
+variables: X0, v1
+absolute: 0.0001
+relative: 0.0001
+amount: X0, v1	
```

Comment:
```suggestion
amount: X0
```
The 'amount' line is supposed to mean 'this is a species, and we want to report its amount instead of its concentration'.  'v1' in this case is not a species.